### DISCUSSION TITLE
Fehler bei Einbindung eines Vektor-TileCaches der basemap

### DISCUSSION THREAD

**Discussion by NoeSaAtz** [e-netze/webgis-community#220] [Q&A]
Hallo Jürgen,
beim Versuch die basemap als Vektor TileCache einzubauen erhalten wir leider eine Fehlermeldung. 

Im CMS wurden beide Schnittstellen (ESRI und Open Source) der Basemap getestet – leider mit identer Fehlermeldung.

Schnittstellen:
Open Source: https://mapsneu.wien.gv.at/basemapvectorneu/root.json
ESRI: https://mapsneu.wien.gv.at/basemapv/bmapv/3857/resources/styles/root.json

Folgende Konfiguration wurde vorgenommen:
Allgemeine Vector Tile Caches > Basemap Vektor > Eigenschaften
![image](https://github.com/user-attachments/assets/c6774e66-e806-4aab-af60-e007e28901ff)
bzw.
![image](https://github.com/user-attachments/assets/3c12afae-d048-4b5e-8c69-0b0ebc98344c)

Fallback-Dienst wurde konfiguriert:
Allgemeine Vector Tile Caches > Basemap Vektor > Fallback
![image](https://github.com/user-attachments/assets/91fcf5ea-d716-484d-bd0c-f0c0ae902d93)

Der Dienst kann im Map Builder ausgewählt werden, es fehlt jedoch das Icon. 
![image](https://github.com/user-attachments/assets/9cb2276a-5af0-496e-b64d-5bc10fba6f24)

Nach Auswahl des Dienstes werden keine Karten mehr im Viewer angezeigt.
![image](https://github.com/user-attachments/assets/565d7e46-360f-4ba6-96b9-8c02398d30a0)

Lg Sandra

**Comment by jugstalt** [comment]
Hallo,
WebGIS verwendet hier zum Darstellen von VTC die MapBox Erweiterungen für Leaflet. Damit kann man "MapBox" Tilecaches einbinden, ob da alles funktioniert kann ich nicht sagen. Getestet hab ich bisher einen Dienst von deinen Kollegen von KAGIS.

Auch andere Dienste wie die Basemap von OpenFreeMap funktionieren https://tiles.openfreemap.org/styles/liberty

<img width="927" alt="image" src="https://github.com/user-attachments/assets/81ff8dc8-a68a-41bd-91bd-20b0ac08077c">

Die Geoland VTC Dienst haben scheinbar ein anderes Format (ESRI?), die mit dem Leaflet Erweiterungen nicht angezeigt werden können.
:(

**Comment by wilhelmberg** [comment]
> Die Geoland VTC Dienst haben scheinbar ein anderes Format (ESRI?), die mit dem Leaflet Erweiterungen nicht angezeigt werden können.
:(

Das ist korrekt.

Die Tileset Definition https://mapsneu.wien.gv.at/basemapv/bmapv/3857/ (verlinkt von https://mapsneu.wien.gv.at/basemapvectorneu/root.json) entspricht nicht der `Mapbox tilejson-spec` https://github.com/mapbox/tilejson-spec

**Reply by jugstalt** [reply]
Danke für den Hinweis Willi :)
Fazit: Die Geolanddienste funktionieren nicht mit WebGIS
